shared/constants/types/wallets.js

```diff
@@ -54,6 +54,7 @@ export type Payment = I.RecordOf<_Payment>
 
 export type _State = {
   accountMap: I.Map<AccountID, Account>,
+  selectedAccount: ?AccountID,
```

Comment:
we might want to use sentries like we do in chat and have a none selected account id. also AccountID should be an opaque type and not a string (i know you didn't write that in this pr but it should be one)

Reply:
Done!